### PR TITLE
[iOS][ColumnSet] ChoiceSet inside ColumnSet has improper layout

### LINKED ISSUE #5399: [iOS][ColumnSet] ChoiceSet inside ColumnSet has improper layout
# Platform

What platform is your issue or question related to? (Delete other platforms).

- [ ] iOS

# Author or host

Are you an author (like sending something to Outlook), or a host that is rendering your own cards? If host, please specify the corresponding host (app) name.

If you're an author, who are you sending cards to?

# Version of SDK

2020.10 (latest master as of Oct 27th)

# Details

When 3 columns is used with ChoiceSet, the layout kindof breaks.

IMG_5469

This can be reproduced using iOS Visualizer app

IMG_0166

The same card however renders fine on Android

Screenshot_20210218-164054

The card JSON for reference
[multi-column-choicesets.txt](https://github.com/microsoft/AdaptiveCards/files/6003474/multi-column-choicesets.txt)

### PR DESCRIPTION
**Pull request by akaashdev**
## Related Issue
Fixed #5399 

## Description
- Overriden the default padding provided by default UITableViewCell
- Set proper frames to subviews in `layoutSubviews` lifecycle method

## Sample Card
```json
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "TextBlock",
            "text": "Choose specific types:",
            "wrap": true
        },
        {
            "type": "ColumnSet",
            "columns": [
                {
                    "type": "Column",
                    "width": "stretch",
                    "items": [
                        {
                            "type": "Input.ChoiceSet",
                            "choices": [
                                {
                                    "title": "Full-time",
                                    "value": "1"
                                },
                                {
                                    "title": "Hourly",
                                    "value": "2"
                                },
                                {
                                    "title": "Agency",
                                    "value": "3"
                                }
                            ],
                            "placeholder": "Please choose employee type (All, Ford, or Agency)",
                            "id": "employee_1",
                            "isMultiSelect": true,
                            "style": "expanded",
                            "value": "0",
                            "wrap": true
                        }
                    ]
                },
                {
                    "type": "Column",
                    "width": "stretch",
                    "items": [
                        {
                            "type": "Input.ChoiceSet",
                            "choices": [
                                {
                                    "title": "Joint Venture",
                                    "value": "4"
                                },
                                {
                                    "title": "Part-time",
                                    "value": "5"
                                },
                                {
                                    "title": "New Employee",
                                    "value": "6"
                                }
                            ],
                            "placeholder": "Please choose employee type",
                            "isMultiSelect": true,
                            "id": "employee_2",
                            "wrap": true
                        }
                    ]
                },
                {
                    "type": "Column",
                    "width": "stretch",
                    "items": [
                        {
                            "type": "Input.ChoiceSet",
                            "choices": [
                                {
                                    "title": "Subsidiary",
                                    "value": "7"
                                },
                                {
                                    "title": "Supplier",
                                    "value": "8"
                                }
                            ],
                            "placeholder": "Placeholder text",
                            "isMultiSelect": true,
                            "id": "employee_3"
                        }
                    ]
                }
            ]
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.3",
    "actions": [
        {
            "type": "Action.Submit",
            "title": "Submit",
            "style": "positive",
            "id": "submit"
        }
    ]
}
```

#### Before fix
Simulator Screen Shot - iPhone 12 Pro Max - 2021-04-20 at 14 03 24

#### After fix
Simulator Screen Shot - iPhone 12 Pro Max - 2021-04-20 at 14 02 09

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5697)

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputChoiceSetRenderer.mm [hunk 1]
```diff
@@ -61,7 +61,7 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
     }
 
     // removes leading padding
-    choiceSetView.contentInset = UIEdgeInsetsMake(0, -15, 0, 0);
+    choiceSetView.contentInset = UIEdgeInsetsMake(0, 0, 0, 0);
```
Comment (jwoo-msft) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputChoiceSetRenderer.mm:64]: nit: can you remove this line?

**Review by jwoo-msft — APPROVED** [review]
:shipit:

**Comment by jwoo-msft** [comment]
@akaashdev 
This change didn't cause layout conflicts, and all frame rectables (UITableCell, UITableView, ImageView and UILabel) are not overlapping, right?

**Comment by akaashdev** [comment]
Yea that's right @jwoo-msft. It didn't cause any layout conflicts as far as I tested and it didn't cause any overlaps.

**Pull request merged**